crates/wallet/src/wallet/coin_selection.rs

```diff
@@ -62,7 +62,7 @@
 //!                 },
 //!             )
 //!             .collect::<Vec<_>>();
-//!         let additional_fees = (fee_rate * additional_weight).to_sat();
+//!         let additional_fees = (fee_rate * additional_weight);
```

Comment:
nit: extra set of round brackets
```suggestion
//!         let additional_fees = fee_rate * additional_weight;
```

Reply:
Fixed.